### ISSUE TITLE
module 'pygame_shaders' has no attribute 'DefaultScreenShader'

### ISSUE THREAD

**Issue by John-Dekka** [ScriptLineStudios/pygame_shaders#9]
Hei there,

just found this module and want to give it a try. But the very first example fails with "module 'pygame_shaders' has no attribute 'DefaultScreenShader'".

What did I do wrong here? :)

**Comment by John-Dekka** [comment]
Oh, nevermind. The package just got updated here. :+1: 

`pip install git+https://github.com/ScriptLineStudios/pygame_shaders.git`


Works fine now. Thanks!

**Comment by ScriptLineStudios** [comment]
Yeah its currently an update in progress, The new versions should be released on pypi soon! In the meantime if you catch any typos in the documentation, please let me know!

**Comment by void4** [comment]
(This is still the case)

**Comment by joereynolds** [comment]
Just tried running it and came across this. The issue shouldn't be closed surely?

**Comment by non-npc** [comment]
same issue here, just installed it using pip and the examples give this error.
testing on windows 10 64bit x86.
